src/helpers/deserializer.test.ts

```diff
@@ -0,0 +1,29 @@
+import { Resource } from 'types/resource';
+
+import { deserialize } from './deserializer';
+
+describe('Deserializer helper', () => {
+  interface TestType extends Resource {
+    name: string;
+    age: number;
+  }
+
+  describe('deserialize', () => {
+    const jsonData = {
+      id: '1',
+      type: 'TestType',
+      attributes: {
+        name: 'Name',
+        age: 25,
+      },
+    };
+
+    it('returns deserialized data', () => {
+      const deserializedData = deserialize<TestType>(jsonData);
+
+      expect(deserializedData.resourceType).toBe('TestType');
+      expect(deserializedData.name).toBe(jsonData.attributes.name);
+      expect(deserializedData.age).toBe(jsonData.attributes.age);
```

Comment:
let's explicitly express what's we are expecting, this way when we change the contents of `jsonData`, we'll have to also change the specs and that's what we want! 🙌🏿 

```suggestion
      expect(deserializedData.name).toBe('Name');
      expect(deserializedData.age).toBe(25);
```

Reply:
@tyrro fixed in [e69d124](https://github.com/manh-t/react-survey/pull/49/commits/e69d1247c0000b2c21c6e8dbf489ddcbdd4d1db6)